gradle.properties

```diff
@@ -0,0 +1,13 @@
+VERSION_NAME=1.1.1-SNAPSHOT
```

Comment:
Maybe start from version 0.0.1-SNAPSHOT ? Or if its ready to roll out 1.0.0-SNAPSHOT ?